controllers/gatekeeper_controller_test.go

```diff
@@ -85,8 +97,11 @@ func TestDeployWebhookConfigs(t *testing.T) {
 	// MutatingWebhookConfiguration disabled
 	gatekeeper.Spec.ValidatingWebhook = &webhookDisabled
 	gatekeeper.Spec.MutatingWebhook = &webhookDisabled
-	deleteAssets, applyAssets = getStaticAssets(gatekeeper)
+	deleteAssets, applyAssets, webhookAssets = getStaticAssets(gatekeeper)
+	g.Expect(webhookAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(webhookAssets).NotTo(ContainElement(MutatingWebhookConfiguration))
 	g.Expect(applyAssets).NotTo(ContainElement(ValidatingWebhookConfiguration))
+	g.Expect(applyAssets).NotTo(ContainElement(MutatingWebhookConfiguration))
 	g.Expect(applyAssets).NotTo(ContainElements(mutatingStaticAssets))
 	g.Expect(deleteAssets).To(ContainElement(ValidatingWebhookConfiguration))
 	g.Expect(deleteAssets).To(ContainElements(mutatingStaticAssets))
```

Comment:
Need a case for `MutatingWebhookConfiguration`

Reply:
Done